http-server/src/main/java/io/airlift/http/server/HttpServerConfig.java

```diff
@@ -385,6 +386,19 @@ public HttpServerConfig setAdminMaxThreads(int adminMaxThreads)
         return this;
     }
 
+    public boolean isUseVirtualThreads()
```

Comment:
I think this needs to be added to `HttpServerBinder` so that the application can choose to use virtual threads, as the application code must be compatible, otherwise we can run out of carrier threads. Or we need to continue to use the configured thread setting for the carrier thread pool. The user configuring the application can't know if the application is virtual thread safe.

Reply:
Done.